### PR TITLE
OSDOCS-2681: Fixes added to #38580

### PR DESCRIPTION
**Pull request by skrthomas**
Original PR created by Mike McKiernan: https://github.com/openshift/openshift-docs/pull/38580

This PR contains fixes and additional information that needs to be added after Mike's departure. 
Support for [MetalLB CR speaker pod tolerations](https://deploy-preview-41167--osdocs.netlify.app/openshift-enterprise/latest/networking/metallb/metallb-operator-install.html#nw-metallb-operator-limit-speaker-to-nodes_metallb-operator-install) is new

### CODE REVIEW

File: modules/nw-metallb-operator-custom-resources.adoc [hunk 1]
```diff
@@ -12,9 +16,32 @@ If the instance is deleted, the Operator removes MetalLB from the cluster.
 MetalLB requires one or more pools of IP addresses that it can assign to a service when you add a service of type `LoadBalancer`.
 When you add an `AddressPool` custom resource to the cluster, the MetalLB Operator configures MetalLB so that it can assign IP addresses from the pool.
 An address pool includes a list of IP addresses.
-The list can be a single IP address, a range specified in CIDR notation, a range specified as a starting and ending address separated by a hyphen, or a combination of the three.
+The list can be a single IP address that is set using a range, such as 1.1.1.1-1.1.1.1, a range specified in CIDR notation, a range specified as a starting and ending address separated by a hyphen, or a combination of the three.
```
Comment (skrthomas) [inline modules/nw-metallb-operator-custom-resources.adoc:19]: @sabinaaledort does this make the single IP address statement accurate?

Comment (sabinaaledort) [reply modules/nw-metallb-operator-custom-resources.adoc:19]: yes, sounds good to me

Comment (skrthomas) [reply modules/nw-metallb-operator-custom-resources.adoc:19]: Ok I'll also make a separate PR for this same fix in 4.9!

File: modules/nw-metallb-bgp.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+* Each `speaker` pod starts a BGP session with all BGP peers and advertises the load balancer IP addresses or aggregated routes to the BGP peers.
+The `speaker` pods advertise that they are part of Autonomous System 65010.
+The diagram shows a router, R1, as a BGP peer within the same Autonomous System.
+However, you can configure MetalLB to start BGP sessions with peers that belong to other Autonomous Systems.
+
+* All the nodes with a `speaker` pod that advertises the load balancer IP address can receive traffic for the service.
+
+** If the external traffic policy for the service is set to `cluster`, then all the `speaker` pods advertise the `203.0.113.200` load balancer IP address and all the nodes with a `speaker` pod can receive traffic for the service.
+At least one endpoint for the service must be in the `Ready` condition.
+
```
Comment (asood-rh) [inline modules/nw-metallb-bgp.adoc:50]: The host prefix is advertised to the peered router only if the external traffic policy is cluster.

File: modules/nw-metallb-bgp-limitations.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+[id="nw-metallb-bgp-limitations-single-asn_{context}"]
+== Support for a single ASN and a single router ID only
+
+When you add a BGP peer custom resource, you specify the `spec.myASN` field to identify the Autonomous System Number (ASN) that MetalLB belongs to.
+{product-title} uses an implementation of BGP with MetalLB that requires MetalLB to belong to a single ASN.
+If you attempt to add a BGP peer and specify a different value for `spec.myASN` than an existing BGP peer custom resource, you receive an error.
+
+Similarly, when you add a BGP peer custom resource, the `spec.routerID` field is optional.
+If you specify a value for this field, you must specify the same value for BGP peer custom resources that you add.
```
Comment (asood-rh) [inline modules/nw-metallb-bgp-limitations.adoc:53]: @skrthomas Could you add bold word in the sentence above to add more clarity?
If you specify a value for this field, you must specify the same value for **all other** BGP peer custom resources that you add.

File: modules/nw-metallb-bgp.adoc (hunk 2 above)
Comment (skrthomas) [reply modules/nw-metallb-bgp.adoc:50]: @asood-rh sounds good, can I say router peer instead of peered router?

**Review by ctauchen — COMMENTED** [review]
@skrthomas I found a couple things, like literally two. LGTM. I was sure I'd found a third, but then it turned out that I just learned a new word: deduplicate. Who knew?

File: networking/metallb/metallb-operator-install.adoc [hunk 4]
```diff
@@ -33,8 +33,7 @@ include::modules/nw-metallb-operator-limit-speaker-to-nodes.adoc[leveloffset=+2]
 .Additional resources
 
 * For more information about node selectors, see xref:../../nodes/scheduling/nodes-scheduler-node-selectors.adoc#nodes-scheduler-node-selectors[Placing pods on specific nodes using node selectors].
-
-[id="next-steps_{context}"]
+* For more information about taints and tolerations, see xref:../../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-scheduler-taints-tolerations-about[Understanding taints and tolerations]
```
Comment (skrthomas) [inline networking/metallb/metallb-operator-install.adoc:36]: @msherif1234 PTAL

File: modules/nw-metallb-operator-limit-speaker-to-nodes.adoc [hunk 5]
```diff
@@ -25,9 +25,14 @@ metadata:
 spec:
   nodeSelector:  <.>
     node-role.kubernetes.io/worker: ""
+  speakerTolerations:
```
Comment (skrthomas) [inline modules/nw-metallb-operator-limit-speaker-to-nodes.adoc:28]: PTAL

File: modules/nw-metallb-operator-limit-speaker-to-nodes.adoc [hunk 6]
```diff
@@ -25,9 +25,15 @@ metadata:
...
+    operator: "Exists"
+    effect: "NoExecute"
 ----
 <.> The example configuration specifies to assign the speaker pods to worker nodes, but you can specify labels that you assigned to nodes or any valid node selector.
+<.> In this example configuration, the pod that this toleration is attached to tolerates any taint that matches the `key` value and `effect` value using the `operator`.
 
 After you apply a manifest with the `spec.nodeSelector` field, you can check the number of pods that the Operator deployed with the `oc get daemonset -n metallb-system speaker` command.
 Similarly, you can display the nodes that match your labels with a command like `oc get nodes -l node-role.kubernetes.io/worker=`.
 
+You can optionally allow the node to control which speaker pods should, or should not, be scheduled on them by applying a list of tolerations. For more information about taints and tolerations, see the additional resrouces.
```
Comment (ahardin-rh) [inline modules/nw-metallb-operator-limit-speaker-to-nodes.adoc:39]: typo in resources

**Comment by ahardin-rh** [comment]
@skrthomas Just one minor comment. Otherwise, the latest changes look good!

**Pull request merged**